planning/constraints.md

```diff
@@ -12,12 +12,10 @@ _Some boundaries around our project._
 
 <!--  constraints that come from within your team, and you have no control over. they may include:  - each of your individual skill levels  - amount of time available to work on the project-->
 
-- We won't use JS since our knowledge is limited.
```

Comment:
Bro this line is not important,  our knowledge is limited I know just don't tell them😂🤣🤣..... Just say js will not be used  😂